File: BackendApi/entities/Advertisement.py
class Advertisement:
    def __init__(self, id, objectId, type, name, status, description, publicationDate, expirationDate):
        self.id = id
        self.objectId = objectId
        self.type = type
        self.name = name
        self.status = status
        self.description = description
        self.publicationDate = publicationDate
        self.expirationDate = expirationDate

    def to_dict(self):
        return {
            "id": self.id,
            "objectId": self.objectId,
            "type": self.type,
            "name": self.name,
            "status": self.status,
            "description": self.description,
            "publicationDate": self.publicationDate,
            "expirationDate": self.expirationDate
        }

File: WoofCare/app/src/main/java/com/example/woofcareapp/api/models/advertisement.kt
package com.example.woofcareapp.api.models

data class Advertisement(
    val id: Int,
    val objectId: Int,
    val type: Int,
    val name: String,
    val status: Int,
    val description: String,
    val publicationDate: String,
    val expirationDate: String
)
